so/challenges/default.json

```diff
@@ -50,4 +50,111 @@
           }
         }
       ]
+    },
+    {
+      "name": "transform",
+      "description": "Run transforms to pivot data by terms and date_histogram",
+      "default": false,
+      "schedule": [
+        {
+          "operation": "delete-index"
+        },
+        {
+          "operation": {
+            "operation-type": "create-index",
+            "settings": {{index_settings | default({}) | tojson}}
+          }
+        },
+        {
+          "name": "check-cluster-health",
+          "operation": {
+            "operation-type": "cluster-health",
+            "index": "logs-*",
```

Comment:
Is it a mistake that we check the health of `logs-*` when the transform operates on the `so` index?

Reply:
https://github.com/elastic/elasticsearch/issues/89980 was filed to address the Cluster Health API behavior when the target does not exist.